project/Dependencies.scala

```diff
@@ -1,14 +1,19 @@
 import sbt.*
 
 object Dependencies {
-
+  val PekkoVersion        = "1.1.0-M1"
   val test: Seq[ModuleID] = Seq(
-    "com.typesafe"         % "config"                  % "1.4.3"  % Test,
-    "com.typesafe.play"   %% "play-ahc-ws-standalone"  % "2.2.7"  % Test,
-    "com.typesafe.play"   %% "play-ws-standalone-json" % "2.2.7"  % Test,
-    "com.vladsch.flexmark" % "flexmark-all"            % "0.64.8" % Test,
-    "org.scalatest"       %% "scalatest"               % "3.2.18" % Test,
-    "org.slf4j"            % "slf4j-simple"            % "2.0.13" % Test
+    "org.apache.pekko"    %% "pekko-actor-typed"         % PekkoVersion,
+    "org.apache.pekko"    %% "pekko-stream"              % PekkoVersion,
+    "org.apache.pekko"    %% "pekko-slf4j"               % PekkoVersion,
+    "org.apache.pekko"    %% "pekko-actor-testkit-typed" % PekkoVersion  % Test,
+    "org.commonmark"       % "commonmark"                % "0.22.0"      % Test,
+    "com.typesafe"         % "config"                    % "1.4.3"       % Test,
+    "org.playframework"   %% "play-ahc-ws-standalone"    % "3.1.0-M2"    % Test,
```

Comment:
We should be using the latest stable version. In this case it's 3.0.5. Same for the one below.